schema/jsonschema.go

```diff
@@ -14,21 +13,17 @@ import (
 	"github.com/santhosh-tekuri/jsonschema/v5/httploader"
 )
 
-const (
-	defaultSchemaURL = "schema.json"
-)
-
-var (
-	//go:embed known_schemas
-	knownSchemas embed.FS
-)
-
 func init() {
 	httploader.Client = &http.Client{
 		Timeout: time.Second * 10,
 	}
 }
 
+const (
+	// defaultSchemaURL is a placeholder that's needed to load any schema
+	defaultSchemaURL = "schema.json"
```

Comment:
Not sure I follow why this is needed, can you elaborate?

Reply:
the library requires you to provide a url for the schema and we don't really care what this is but it needs to be a placeholder https://github.com/santhosh-tekuri/jsonschema/blob/master/compiler.go#L65